config.yaml

```diff
@@ -0,0 +1,2 @@
+Google:
+    SheetID: 1httLutCH3tj1G8aM5QjeklfvwPsOAsLGulWAQ4g8LGo
```

Comment:
Similarly it looks like this is being pulled from the `environment.ts` file in the code. Was this an accidental commit along with the secrets example?

Reply:
I left the config file in there so anyone else who builds on this project would have an easier time setting up the environment since they would just need to update the two configs and `docker compose up`